.jenkins/pytorch/build.sh

```diff
@@ -88,11 +88,3 @@ if [[ "$BUILD_ENVIRONMENT" == *xenial-cuda8-cudnn6-py3* ]]; then
   make html
   popd
 fi
-
-# Test no-Python build
```

Comment:
We don't need this anymore because the normal `setup.py` will build this anyway, correct?

Reply:
yes